sdk/node/test/accounts.js

```diff
@@ -0,0 +1,246 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+const signer = new chain.HsmSigner()
+
+const aliceAlias = `alice-${uuid.v4()}`
+const bobAlias = `bob-${uuid.v4()}`
+
+let aliceKey, bobKey, otherKey
+
+describe('Accounts test', () => {
+
+  describe('Promise style', () => {
+
+    before('set up API objects', () => {
+
+      // Key creation and signer setup
+      return expect(Promise.all([
+        client.mockHsm.keys.create({alias: aliceAlias}),
+        client.mockHsm.keys.create({alias: bobAlias}),
+        client.mockHsm.keys.create()
+      ])).to.be.fulfilled
+
+      .then(keys => {
+        aliceKey = keys[0]
+        bobKey = keys[1]
+        otherKey = keys[2]
```

Comment:
Could we get away with just one mock HSM key for these tests? 3 seems excessive.